solr/core/src/test-files/solr/solr.xml

```diff
@@ -40,12 +40,12 @@
     <str name="host">127.0.0.1</str>
     <int name="hostPort">${hostPort:8983}</int>
     <str name="hostContext">${hostContext:solr}</str>
-    <int name="zkClientTimeout">${solr.zkclienttimeout:30000}</int>
+    <int name="zkClientTimeout">${solr.zkclienttimeout:45000}</int> <!-- This should be high by default - dc's are expensive -->
     <bool name="genericCoreNodeNames">${genericCoreNodeNames:true}</bool>
-    <int name="leaderVoteWait">${leaderVoteWait:10000}</int>
-    <int name="leaderConflictResolveWait">${leaderConflictResolveWait:180000}</int>
-    <int name="distribUpdateConnTimeout">${distribUpdateConnTimeout:45000}</int>
-    <int name="distribUpdateSoTimeout">${distribUpdateSoTimeout:340000}</int>
+    <int name="leaderVoteWait">${leaderVoteWait:15000}</int>   <!-- We are running tests - the default should be low, not like production -->
```

Comment:
The older explicitly set value for  leaderVoteWait was 10000 and now it's increased to 15000 but the comment seems to indicate that we wanted to lower it.

Just wanted to make sure it isn't a typo

Reply:
No, didn't necessarily want to lower it, just explain why it's set so low compared to what would be recommended in production (minutes).